packages/device_info/lib/device_info.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
We're switching plugins to use class containers to make them easier to mock - including having a `channel` property to tests can set the channel to use fakes.

e.g. https://github.com/flutter/plugins/blob/master/packages/google_sign_in/lib/google_sign_in.dart#L102